example-client/client.py

```diff
@@ -10,10 +13,30 @@
     print_streaming_events,
     get_structure_run_logs,
 )
-from requests.exceptions import HTTPError
 
+# Pull in configuration variables.
+load_dotenv()
+
+# If running against the Skatepark emulator, you will need to set
+# this environment variable (or put it in a .env file) to tell the
+# client which URL Skatepark is listening on. The default value is
+# http://127.0.0.1:5000 .
+# For a Structure in Griptape Cloud, this environment variable 
+# should be set to https://cloud.griptape.ai/
+HOST = os.environ["GT_CLOUD_BASE_URL"]
+
+# If running against the Skatepark emulator, this will be the ID
+# of the Structure that you registered with Skatepark. For a Structure
+# hosted in Griptape Cloud, it will be the ID of the Structure you 
+# are running, which can be found at https://cloud.griptape.ai/structures 
 GT_STRUCTURE_ID = os.environ["GT_STRUCTURE_ID"]
 
+# The Skatepark emulator does NOT require a Griptape API key.
+# However, Griptape Cloud requires a valid API Key in order
+# to authorize calling the API. You can generate a key by
+# visiting https://cloud.griptape.ai/keys
+GT_API_KEY = os.environ.get("GT_CLOUD_API_KEY", "GRIPTAPE API KEY ONLY NEEDED FOR STRUCTURES IN GRIPTAPE CLOUD")
```

Comment:
With the `.env` examples do we need to provide a fallback value here? I think I'd rather it error out indicating that the value is missing and their `.env` configuration is wrong.

Reply:
We do warn the user if they don't specify one. And Jason thought it was "fancy." I could go either way.